src/Akeneo/Category/back/Infrastructure/Storage/Save/Query/UpsertCategoryBaseSql.php

```diff
@@ -51,9 +51,9 @@ private function insertCategory(Category $categoryModel): void
             ;
         SQL;
 
-        $values = $categoryModel->getAttributes();
-        if (null !== $values) {
-            $values = json_encode($values->normalize());
+        $attributes = $categoryModel->getAttributes();
+        if (null !== $attributes) {
+            $attributes = json_encode($attributes->getValues());
         }
```

Comment:
For a good understanding, we should differentiate the naming we use for the attribute entities and their values. 
What do you think about using `attributeValues` here?  (@akeneo/squad-griffins) 
```suggestion
        $attributes = $categoryModel->getAttributes();
        $attributeValues = '';
        if (null !== $attributes) {
            $attributeValues = json_encode($attributes->getValues());
        }
```

Reply:
Done